selfdrive/car/chrysler/values.py

```diff
@@ -110,6 +111,21 @@ class ChryslerCarInfo(CarInfo):
   ],
 }
 
+FW_VERSIONS: Dict[str, Dict[Tuple[capnp.lib.capnp._EnumModule, int, Optional[int]], List[str]]] = {
+  CAR.RAM_1500: {
+    (Ecu.combinationMeter, 0x742, None): [],
```

Comment:
After release we'll want to do some analysis to figure out which of these duplicate Ecu types are common across all cars (original PR had one fwdCamera on Chrysler, one of Jeep/Ram for example, but Ram responds to both)